nav2_dwb_controller/dwb_core/include/dwb_core/trajectory_critic.hpp

```diff
@@ -166,7 +167,8 @@ class TrajectoryCritic
    *
    * @param pc PointCloud to add channels to
    */
-  virtual void addCriticVisualization(sensor_msgs::msg::PointCloud &) {}
+  // FIXME(sachin): is there a compultion to use ros type messages as arguments ?
```

Comment:
Please resolve, ask, or remove fixme's

Reply:
Sorry. My bad. I had removed them. Might have reset somewhere and lost those changes. will make sure to remove them.